sync/v1.proto

```diff
@@ -132,53 +184,79 @@ enum FileAccessAction {
 message PreflightResponse {
   // The client mode that the client should move into at the end of this sync.
   // The mode does not change until the Postflight request has been made, to
-  // ensure the client has received all the needed rules before potentially going
-  // into Lockdown.
+  // ensure the client has received all the needed rules before potentially
+  // going into Lockdown.
   ClientMode client_mode = 1;
 
+  // The SyncType of this sync, which impacts how rules are applied. See the
+  // comments on the `SyncType` enum to learn how Santa interprets this value.
   optional SyncType sync_type = 2;
 
   // Controls how many events Santa should upload in a single EventUpload request.
   // If the server doesn't specify, the default is 50.
   uint32 batch_size = 3;
 
-  // Enable bundle hashing and bundle rules.
+  // Enable bundle hashing and bundle rules. If this is enabled when a binary
+  // is blocked Santa will try to determine the outermost bundle that it is a
+  // part of and create a "bundle hash" for the whole bundle. When the sync
+  // server receives this bundle hash it can decide whether to request that
+  // Santa uploads more details about all of the binaries contained within the
+  // bundle.
   optional bool enable_bundles = 4;
 
   // Enable transitive (ALLOWLIST_COMPILER) rules.
-  // Without this enabled, any received ALLOWLIST_COMPILER rules will be treated as ALLOWLIST.
+  // Without this enabled, any received ALLOWLIST_COMPILER rules will be treated
+  // as ALLOWLIST.
   optional bool enable_transitive_rules = 5;
 
-  // Ordinarily, Santa will only upload events about executions that are denied or would be denied if the machine
-  // were in LOCKDOWN mode. With this enabled, Santa will upload details about all events.
+  // Ordinarily, Santa will only upload events about executions that are denied
+  // or would be denied if the machine were in LOCKDOWN mode. With this enabled,
+  // Santa will upload details about all events.
```

Comment:
nit: might be good to somehow callout these are not 1-to-1 with execs and multiple layers of caching means not every exec is logged.